template/.github/workflows/codecov.yml.jinja

```diff
@@ -0,0 +1,43 @@
+name: Codecov
```

Comment:
Where would be the best place to notify that this works for public repos? I was thinking on the checklist.

Reply:
Just so we don't miss the idea here, do you want to add something in the checklist in this PR, or would like me to open an issue so we don't forget?